variables.tf

```diff
@@ -0,0 +1,110 @@
+variable "cluster_arn" {
+  type        = string
+  description = "The ARN of the ECS cluster"
+  default     = null
+}
+
+variable "config" {
+  description = "A map containing all labeler configuration options"
```

Comment:
are these variables described somewhere? Can we link to the energylabeler documentation? For some its hard to know what can be entered.

Reply:
it's even hard to find with the documentation.  I have to go into the lib from the source of the cli to find any of it :S

asked the maintainers to update the documentation but can provide a link to the readme

Reply:
https://github.com/schubergphilis/terraform-aws-mcaf-energy-labeler/blob/13452ff07183a176e80e2dccdc58f7182ecb866f/README.md#terraform-aws-mcaf-energy-labeler

first documentation links to the library it's based on.